include/dl/dl.h

```diff
@@ -411,6 +411,70 @@ typedef struct dl_instance_info
 */
 dl_error_t DL_DLL_EXPORT dl_instance_get_info( const unsigned char* packed_instance, size_t packed_instance_size, dl_instance_info_t* out_info );
 
+/*
+	Function: dl_get_metadata_cnt
+	    Fetch the number of meta data instances for the specified type, enum or union
+
+	Parameters:
+	    dl_ctx          - Context to load type-library into.
+	    type            - Type id for type to fetch the meta data count of.
+	    count           - The number of meta data instances available for the give type
+
+	Return:
+	    DL_ERROR_OK on success.
+*/
+dl_error_t DL_DLL_EXPORT dl_get_metadata_cnt( dl_ctx_t dl_ctx, dl_typeid_t type, unsigned int* count );
```

Comment:
have I used _cnt in the public api before? I think I would prefer _count() for public api.

Reply:
Yes, in DL_TYPE_ATOM_CNT & DL_TYPE_STORAGE_CNT, and in a bunch of private places. But I agree, it could sound a bit like genitalia